metadata/contributors.json

```diff
@@ -7474,13 +7475,6 @@
          "yuzo"
       ]
    },
-   {
-      "emails" : [
-         "yvlai@apple.com"
-      ],
-      "github" : "yvonne-lai1",
-      "name" : "Yvonne Lai"
-   },
```

Comment:
Why was this removed as part of this PR @marcoscaceres ?

Reply:
CC @yvonne-lai1

Reply:
No idea what happened. Sending a PR t fix this. Massive apologies to @yvonne-lai1 for my mistake here. I'll send another PR to fix this straight away.